packages/mocks/src/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from './definitions/basic/index';
+export * from './definitions/resolvePath';
```

Comment:
this should be between the two comments

Reply:
this is manually created - why  should it go under generated section?
btw what will happen after running the tool on this PR? will all `*.ts` files will be listed here?

Reply:
the tool does not run by itself, you need to manually run it when you want to add exports, it's just easier to run it then type in all the files, sometimes it can be a lot of files, and running a tool is easier ant prevent typos

Reply:
for example here, I think it was easier to run the tool, then to write this exports manually